namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-audit-dev/04-networkpolicy.yaml

```diff
@@ -24,4 +24,4 @@ spec:
     - from:
         - namespaceSelector:
             matchLabels:
-              component: ingress-controllers
+              component: hmpps-audit-dev
```

Comment:
This would stop access the auth url from the browser or from any other service. Can you create a new network policy and allow a target-namespace. Also the label should be: `cloud-platform.justice.gov.uk/namespace: "source-namespace'` where the source-namespace is from where the request is coming from.